src/ngraph_builder.cc

```diff
@@ -1422,8 +1422,8 @@ static Status TranslateFusedBatchNormGradOp(
   TF_RETURN_IF_ERROR(ValidateInputCount(op, 5));
 
   bool tf_is_training;
-  // Now we only support is_training=true case. We will either mark
-  // or implement is_training=false case later.
+  // Now we only support is_training=true case. We marked rejection for the case
```

Comment:
Is this comment accurate i.e., is it that `we only support is_training=true case` now or is this permanent?

Reply:
Removed `Now` in the comment